src/server/api/routers/generations/getAllQuery.ts

```diff
@@ -23,6 +23,8 @@ export const getAllQuery = protectedProjectProcedure
         traceId: string;
         traceName: string;
         latency: number | null;
+        promptName: string | null;
+        promptVersion: string | null;
```

Comment:
This should be always null, as we only have the `promptId` on the observations_view, right?

Reply:
Due to the left join in the rawSqlQuery coming from getAllGenerationsSqlQuery we have both properties available. Therefore we need to extend the type. The promptId is available via the ObservationView Type.